imgui-sys/README.md

```diff
@@ -0,0 +1,96 @@
+# Updating to new imgui versions
```

Comment:
I'd prefer if docs like this should probably be in `<project-root>/docs/updating.md`.

One reason not to use the readme is that it will show up on crates.io, and so if we have one (which we should, tbh, but probably not as part of this changeset) it would be better for that to be more user-facing.

Reply:
Ohh, yes, I completely forgot imgui-sys gets published like that! Shall move update and move this doc to that folder, and maybe put a placeholder imgui-sys README in its place